TestFoundation/TestFileManager.swift

```diff
@@ -447,13 +447,25 @@ class TestFileManager : XCTestCase {
             var foundItems = Set<String>()
             while let item = e.nextObject() as? String {
                 foundItems.insert(item)
+#if os(Windows)
+                if item == "item" {
+                    item1FileAttributes = e.fileAttributes
+                } else if item == "path2\\item" {
+                    item2FileAttributes = e.fileAttributes
+                }
+#else
                 if item == "item" {
                     item1FileAttributes = e.fileAttributes
                 } else if item == "path2/item" {
                     item2FileAttributes = e.fileAttributes
                 }
+#endif
             }
+#if os(Windows)
+            XCTAssertEqual(foundItems, Set(["item", "path2", "path2\\item"]))
```

Comment:
I wonder if you can span the `Set` over two lines and do an `#if os(Windows)` to swap out the third element.

Reply:
You can't. You can move the set creation elsewhere, or make it mutable and `#if os(Windows)` the addition.